cpp/src/io/orc/reader_impl.cu

```diff
@@ -1101,6 +1105,7 @@ table_with_metadata reader::impl::read(size_type skip_rows,
     // Association between each ORC column and its cudf::column
     _col_meta.orc_col_map.emplace_back(_metadata->get_num_cols(), -1);
     std::vector<orc_column_meta> nested_col;
+    bool is_stream_empty = false;
```

Comment:
does not seem like the right name - this refers to multiple streams, does it not?

Reply:
Yeah, may be I can change it to `is_data_empty` or `are_streams_empty`